### DISCUSSION TITLE
firecfg: support adding more applications

### DISCUSSION THREAD

**Discussion by ghost** [netblue30/firejail#6273] [Ideas]
Now we have support for `disabling` apps from fircfg via `!foo` IMO it would make sense to allow `enabling` apps too. Syntax could be `+foo` or whatever, that's always best left to the implementer(s).

Let me give an example why I think this is something worthwhile to consider.
Currently we have 23 apps (archivers) that are not enabled by default because of a needed fix for makepkg on Arch Linux. That's a wise thing to do. But it negatively affects users on other OSes. Those users could benefit from a way to allow those.

Besides the above, it's always possible that users have custom profiles for custom apps that aren't in our distributed set of profiles. They could benefit from an easy way to add firecfg support too.

**Comment by kmk3** [comment]
> Now we have support for `disabling` apps from fircfg via `!foo` IMO it would
> make sense to allow `enabling` apps too. Syntax could be `+foo` or whatever,
> that's always best left to the implementer(s).

You can add `foo` to `/etc/firejail/firecfg.d/bar.conf`.

The syntax for firecfg.config and .conf files is the same.

Relates to:

* #5876
